test/auth.e2e-spec.ts

```diff
@@ -0,0 +1,279 @@
+import * as request from 'supertest';
+import { Test, TestingModule } from '@nestjs/testing';
+// import { AuthModule } from '../auth/auth.module';
+
+import { AuthService } from '../src/auth/auth.service';
+import { User } from '../src/model/user.model';
+import { UserGender, UserRole, UserStatus } from '../src/enum/user.enum';
+
+let app = 'http://localhost:3000';
+const loginPayload = {
+    username: 'admin1',
+    password: 'admin1',
+};
+describe('Auth', () => {
+    beforeAll(async () => {
+        await request(app)
+            .post(`/auth/login/`)
+            .send(loginPayload)
+            .then(async (res) => {
+                let token = res.text;
+                await request(app)
+                    .delete(`/admin/deleteUser`)
+                    .set('Authorization', 'Bearer ' + token)
+                    .send({ username: 'B-sh-X0' });
+                await request(app)
+                    .delete(`/admin/deleteUser`)
+                    .set('Authorization', 'Bearer ' + token)
+                    .send({ username: 'B-sh-X1' });
+            });
+    });
+
+    it('Testcase TCA1 should be invalid : username already exists', async () => {
+        const user: User = {
+            username: 'admin1',
+            password: 'bruno1K258',
+            email: 'b2b@mail.com',
+            firstName: 'Nonthanat',
+            lastName: 'Theeratanapartkul',
+            address: 'Moo 5 Ban Kra Bye',
+            birthDate: new Date(),
+            ssin: 'G12NS38N04W9782',
+            gender: UserGender.Male,
+            role: UserRole.Tutor,
+            status: UserStatus.Active,
+            credit: 0,
+            verified: false,
+        };
+        return await request(app)
+            .post('/auth/register')
+            .send(user)
+            .expect(({ body }) => {
+                // console.log(body)
+            })
```

Comment:
What does this mean?

Reply:
forget to remove it

Reply:
don't care this e2e, it's not the last version